packages/api/src/cli/codegen/languages/typescript.ts

```diff
@@ -672,14 +679,15 @@ sdk.server('https://eu.api.example.com/v14');`)
    */
   prepareParameterTypesForOperation(operation: Operation, operationId: string) {
     const schemas = operation.getParametersAsJSONSchema({
+      includeDiscriminatorMappingRefs: false,
```

Comment:
I added this new option in https://github.com/readmeio/oas/pull/698. With it turned off any discriminator mapping that is a ref pointer won't automatically include the components for that ref pointer alongside `discriminator.mapping`. We normally have this happen because our form system in the API Explorer needs it to be colocated like that but I don't need that logic here, and having those components there was causing lots of problems with the `transformer` work I have a few lines down from here transforming dereferenced `$ref` schemas into primitive strings -- that we later use to correlate them to generated TS types.

Codegeneration is a goddamn hard problem and it's been a _week_.